website/pages/docs/commands/monitor.mdx

```diff
@@ -0,0 +1,44 @@
+---
+layout: docs
+page_title: monitor - Command
+sidebar_title: <code>monitor</code>
+description: |-
+  The "monitor" command displays the server logs of a Vault server in real
+  time.
+---
+
+# monitor
+
+The `monitor` command shows a real time display of the server logs of a Vault
+server. This command accepts a log level as an argument, which can be different
+from the log level that the Vault server was started with.
+
+The `monitor` command honors the same the variables that the base command
```

Comment:
I don't know that we need to call out standard behaviour with respect to tokens.  I do think it makes sense to keep the bit about VAULT_ADDR, since it's relatively unusual that we're behaving differently depending on which node you connect to.

Reply:
I looked at the docs for a few different commands to get a sense for how our docs are written before writing this up. That particular piece was in the docs for `vault debug` which is how it got included here. I'll update that part.